pkg/api/types.go

```diff
@@ -1595,6 +1595,45 @@ type Handler struct {
 	TCPSocket *TCPSocketAction
 }
 
+// DeleteExecAction describes a "run in container" action that will be invoked
```

Comment:
Since the type is DeleteExecAction do you want to say that it describes an "exec in container" action?